src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -20,6 +20,27 @@
 namespace Microsoft.PowerShell.Commands
 {
     /// <summary>
+    /// Exception class for webcmdlets to enable returning HTTP error response
+    /// </summary>    
+    public sealed class HttpResponseException : Exception
```

Comment:
I suggest we make `HttpResponseException` derive from `HttpRequestException`, so that `{ Invoke-WebRequest <url> } catch [HttpRequestException] { }` will continue to work after this change.